packages/components/inputs/time-input/src/time-input.spec.js

```diff
@@ -97,6 +97,24 @@ describe('TimeInput', () => {
     );
   });
 
+  it('should call onChange when clearing the value', () => {
```

Comment:
There was no test for clearing the value, so the regression slipped through unfortunately.